compiler/src/iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h

```diff
@@ -21,6 +21,9 @@
 #include "mlir/Interfaces/FunctionInterfaces.h"
 
 namespace mlir::iree_compiler {
+
+const std::string kIreeCodegenUkernelAttrName = "iree_codegen.ukernel";
```

Comment:
```suggestion
constexpr char kCodegenUkernelAttrName[] = "iree_codegen.ukernel";
```
Use char[] instead of std::string and I think `kIree` isn't really used in the codebase/standardized on.